ROADMAP.md

```diff
@@ -71,9 +71,23 @@ A package manager looking to adopt Sigstore as part of its artifact signing and
 
 The npm registry, the world's largest open source package manager for the JavaScript ecosystem, [recently announced built-in support for Sigstore-signed SLSA build provenance](https://github.blog/2023-04-19-introducing-npm-package-provenance/) to users. This allows consumers of packages from npm to independently verify the link between a package fetched from the npm registry to the source code from which it was built. The build provenance attestation is cryptographically signed using Sigstore. Other package managers may find the [documentation of npm's Sigstore adoption and resulting design decisions](https://repos.openssf.org/build-provenance-for-all-package-registries) useful.
 
```

Comment:
Can we add a paragraph about GitHub artifact attestations before Homebrew, and then mention homebrew leverages that?